config/software/datadog-pip.rb

```diff
@@ -0,0 +1,33 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2018 Datadog, Inc.
+
+name "datadog-pip"
+
+dependency "pip"
+
+source git: "https://github.com/DataDog/pip.git"
+
+relative_path "pip"
+
+pip_version = ENV["PIP_VERSION"]
```

Comment:
is this env var something we should also add to `datadog-agent`'s `release.json` file? Or for now we can just use the tip of `trishankatdatadog/10.0.1.tuf`?

Reply:
I added the env var with that in mind, but since there's still some development going on (and I'm uncertain of what's going to happen) I think we can leave it pinned.